src/index.html

```diff
@@ -1,59 +1,61 @@
 <!doctype html>
-<html lang="en">
+<html lang='en'>
```

Comment:
Are you sure, that it is a best practice to have single quotes in HTML? Please research.